README.md

```diff
@@ -336,7 +343,7 @@ For example:
 const serve = require('webpack-serve');
 const config = require('./webpack.config.js');
 
-serve({ config }).then((server) => {
+serve({ config }).then(server => {
```

Comment:
> I accidentaly ran prettier on the readme.md but the changes seemed good so I just stuck with it. (sorry for the random changes is what I'm trying to say).

Maybe (especially since i mentioned this in my pull request) there is a nicer way to phrase this and not spam me with the same message about something that is not my personal preference at all (as a matter of fact I didn't realize it happened until I started writing the pr message).